rules/legacy_xcodeproj.bzl

```diff
@@ -1312,6 +1315,7 @@ Additional LLDB settings to be added in each target's .lldbinit configuration fi
         "bazel_profile_enabled": attr.bool(default = False, mandatory = False),
         "disable_main_thread_checker": attr.bool(default = False, mandatory = False),
         "force_x86_sim": attr.bool(default = False, mandatory = False),
+        "application_env_vars": attr.string_dict(mandatory = False, default = {}, doc = "Add env vars to launch actions for app schemes")
```

Comment:
Any reason not to use `scheme_existing_envvar_overrides` - it seems like if we add it for `application` it'll be a slippery slope - for every potential kind of bundle. 

FWIW XCHammer will allow you configure such options at the _target level_ - and have such [scheme options applying to a given target](https://github.com/pinterest/xchammer/blob/master/BazelExtensions/xchammerconfig.bzl#L47)

Reply:
Also, I'm wondering if it'd be better to add it to the point which allows _target_ config:
https://github.com/bazel-ios/rules_ios/blob/master/tests/ios/xcodeproj/BUILD.bazel#L32

Reply:
I think the scheme_existing_envvar_overrides requires there to be an env_vars kwarg in the target already.  I found including scheme_existing_envvar_overrides yields nothing by itself, and then when adding env_vars to ios_application the attribute is passed down to _objc and _swift rules which don't handle it.

I'll check out the target level config.  Are you thinking it would be another attribute like `app_environment` analogous to the existing `test_environment`?